source/guide_littlelinkcustom.rst

```diff
@@ -0,0 +1,110 @@
+.. highlight:: console
+
+.. author:: Jan Klomp <https://klomp.de>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: lang-php
+.. tag:: web
+.. tag:: self-hosting
+.. tag:: analytics
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/littlelinkcustom.svg
+      :align: center
+
+##########
+LittleLink Custom
+##########
+
+.. tag_list::
+
+LittleLinkCustom_ is a highly customizable link sharing and landing page platform with an easy to use interface and acts as a free opensource alternative to Linktree, Shorby, bio.fm etc.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+    * :manual:`PHP <lang-php>`
+    * :manual:`domains <web-domains>`
+
+License
+=======
+
+*LittleLink Custom* is licensed under the GNU General Public License v3.0. All relevant legal information can be found here:
+
+  * https://github.com/JulianPrieber/littlelink-custom/blob/main/LICENSE
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.4:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.4'
```

Comment:
Hi 👋,
since PHP 7.4 is nearing its EOL, have you tried version 8.0 or 8.1?

Reply:
Hello!

Thank you! Yes, I've tested the installation and functionality with PHP 8.1 and it works fine.

I've updated the guide now.